site/_includes/header.html

```diff
@@ -98,7 +98,7 @@
                       <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Documentation</a>
                       <ul class="dropdown-menu">
                         <li><a href="docs/home">Documentation Home</a></li>
-                        <li><a href="{{ site.projurl }}/docs/javadoc/index.html">Javadoc</a></li>
```

Comment:
The original used `{{ site.projurl }}` to provide a level of indirection.  That seems generally good and particularly desirable since the url will change once again when the project name is changed.  Can such a scheme be applied everywhere - at least for non-autogenerated javadoc files?  Related question: can something similar  be done for the project name - e.g., something like `site.projname` instead of using `Quarks` directly?